sml_small/editing/totals_and_components/example.py

```diff
@@ -222,10 +223,10 @@ def invoke_process_with_local_csv():
                 (row["reference"]),
                 (row["total"]),
                 [
-                    (row["comp_1"]),
-                    (row["comp_2"]),
-                    (row["comp_3"]),
-                    (row["comp_4"]),
+                    float('Nan') if not row["comp_1"] else (row["comp_1"]),
```

Comment:
You have not done the same checks in the _invoke_process_with_in_memory_csv_. That function does have hard-coded in-memory CSV data but it may be good to keep it consistent.

Reply:
Updated